spec/requests/api/v1/provider_events_spec.rb

```diff
@@ -37,14 +37,19 @@
     let(:parsed_response) { JSON.parse(response.body) }
 
     context "when authorized" do
+      let(:parsed_response) { JSON.parse(response.body) }
+
       before do
         default_headers[:Authorization] = bearer_token
         default_headers[:CONTENT_TYPE] = "application/json"
       end
 
-      it "returns 204 status when successful" do
-        post "/api/v1/participant-declarations", params: params.to_json
-        expect(response.status).to eq 204
+      it "create declaration record and return id when successful" do
+        expect {
+          post "/api/v1/participant-declarations", params: params.to_json
+        }.to change(ParticipantDeclaration, :count).by(1)
+        expect(response.status).to eq 200
+        expect(parsed_response["id"]).to eq(ParticipantDeclaration.last.id)
```

Comment:
`ParticipantDeclaration .order(:created_at).last` please, `last` the default db order is... alphabetical order of uids (I think).

Reply:
Done, thanks for that one. Much more robust.